pdr_backend/cli/cli_arguments.py

```diff
@@ -112,6 +115,43 @@ def add_argument_LOOKBACK(self):
         )
 
 
+@enforce_types
+class NUM_Mixin:
+    def add_argument_NUM(self):
+        self.add_argument("NUM", type=int, help="1|2|..")
+
+
+@enforce_types
+class ACCOUNTS_Mixin:
+    def add_argument_ACCOUNTS(self):
+        self.add_argument(
+            "ACCOUNTS",
+            type=str,
+            help="Comma-separated list of accounts",
+        )
+
+
+@enforce_types
+class TOKEN_AMOUNT_Mixin:
+    def add_argument_TOKEN_AMOUNT(self):
+        self.add_argument(
+            "TOKEN_AMOUNT",
+            type=float,
+            help="Amount of token to send to each address",
+        )
+
+
+@enforce_types
+class NATIVE_TOKEN_Mixin:
+    def add_argument_NATIVE_TOKEN(self):
+        self.add_argument(
+            "--NATIVE_TOKEN",
+            action="store_true",
+            default=False,
+            help="Is native token or OCEAN?",
```

Comment:
please make sure the help and type of these indicate it is a boolean. It is unclear to me if both native and ocean give "True". I suggest something like one of these a help string:
- "True if native token or OCEAN, False otherwise. Defaults to false"
- "True if native token, as opposed to OCEAN. Defaults to false (OCEAN)"

Reply:
Great feedback. I've updated the help comments. However, `store_true` & `default` is how you use booleans in python argparse. So, I kept it implemented like this, with the new comments and usage in place.

Here is the problem w/ using `type=bool` as described by SO
```
What I find misleading is that you can set type=bool, you get no error message, and yet, for both "False" and "True" string arguments, you get True in your supposedly boolean variable (due to how type casting works in python). So either type=bool should be clearly unsupported (emit some warning, error, etc.), or it should work in a way that is useful and intuitively expected.
```

https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse